pygluu-compose/pygluu/compose/app.py

```diff
@@ -30,8 +31,20 @@
 )
 
 
-class ContainerHelper:
+PROJ_DIR = os.path.abspath(os.path.dirname(sys.argv[0]))
```

Comment:
The `PROJ_DIR` path will be resolved to the location of where `pygluu-compose` executable is installed to, i.e. `$HOME/.virtualenv/pygluu-compose/bin` instead of current working directory.

Reply:
As a result, the `docker.env` seems not loaded because the base directory is not the current working directory.
To reproduce the issue, run the following commands:

```
cd /path/to/working/directory
pygluu-compose init
pygluu-compose up
```

This is the output from last command:

```
The GLUU_JACKRABBIT_NAME variable is not set. Defaulting to a blank string.
The GLUU_OXAUTH_NAME variable is not set. Defaulting to a blank string.
The GLUU_OXTRUST_NAME variable is not set. Defaulting to a blank string.
The GLUU_PERSISTANCE_NAME variable is not set. Defaulting to a blank string.
The GLUU_CONFIGURATION_NAME variable is not set. Defaulting to a blank string.
Creating pull-33_vault_1  ... done
[I] Checking Vault status
Traceback (most recent call last):
  File "/home/iromli/.virtualenv/pygluu-compose/lib/python3.6/site-packages/docker-4.4.4-py3.6.egg/docker/api/client.py", line 268, in _raise_for_status
    response.raise_for_status()
  File "/home/iromli/.virtualenv/pygluu-compose/lib/python3.6/site-packages/requests-2.26.0-py3.6.egg/requests/models.py", line 953, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: http+docker://localhost/v1.40/containers/vault/exec
```